extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -43,6 +44,20 @@ const query_target_databases_sql = `
 		AND is_distributor <> 1
 	ORDER BY db.name;`;
 
+const query_databases_with_size = `
+	WITH
+		db_size
+		AS
+		(
+			SELECT database_id, CAST(SUM(size) * 8.0 / 1024 AS INTEGER) size
+			FROM sys.master_files with (nolock)
+			GROUP BY database_id
+		)
+	SELECT name, state_desc AS state, db_size.size
+	FROM sys.databases with (nolock) LEFT JOIN db_size ON sys.databases.database_id = db_size.database_id
+	WHERE sys.databases.state = 0
+	`;
+
 export const excludeDatabses: string[] = [
```

Comment:
nit: if possible, would you be able to fix the typo here as well

Reply:
fixed